Gale Databases.js

```diff
@@ -1,190 +1,211 @@
-	var post = "citationFormat=RIS&documentData=" + encodeURIComponent(documentData).replace(/%20/g, "+");
-	var pdfurl = attr(doc, '#docTools-pdf a', 'href');
-
-	// Z.debug(post)
-	ZU.doPost(postURL, post, function(text){
-
-		text = text.trim();
-		// gale puts issue numbers in M1
-		text = text.replace(/M1\s*\-/g, "IS  -");
-		// L2 is probably meant to be UR, but we can ignore it altogether
-		text = text.replace(/^L2\s+-.+\n/gm, '');
-		// we can map copyright notes via CR
-		text = text.replace(/^N1(?=\s+-\s+copyright)/igm, 'CR');
-		// Z.debug(text);
-		
-		var translator = Zotero.loadTranslator("import");
-		translator.setTranslator("32d59d2d-b65a-4da4-b0a3-bdd3cfb979e7");
-		translator.setString(text);
-		translator.setHandler("itemDone", function (obj, item) {
-			if (item.ISSN) {
-				item.ISSN = ZU.cleanISSN(item.ISSN);
-			}
-			if (item.pages && item.pages.endsWith("+")) {
-				item.pages = item.pages.replace(/\+/, "-");
-			}
-			if (pdfurl) {
-				item.attachments.push({
-					url: pdfurl,
-					title: "Full Text PDF",
-					mimeType:'application/pdf'
-				});
-			} else {
-				item.attachments.push({document: doc, title: "Snapshot"});
-			}
-			item.complete();
-		});
-		translator.translate();
+function translate(data) {
+	var translator = Zotero.loadTranslator("import");
+	translator.setTranslator(DATA_TRANSLATOR);
+	translator.setString(transform(data));
+	translator.setHandler("itemDone", function (obj, item) {
+		if (item.ISSN) {
+			item.ISSN = Zotero.Utilities.cleanISSN(item.ISSN);
+		}
+		if (item.pages && item.pages.endsWith("+")) {
+			item.pages = item.pages.replace(/\+/, "-");
+		}
+		item.attachments.push({ document: data, title: "Snapshot" });
+		item.complete();
 	});
+	translator.translate();
 }
 
-function doWeb(doc, url) {
-	if (detectWeb(doc, url) == "multiple") {
-		Zotero.selectItems(getSearchResults(doc, false), function (items) {
-			if (!items) {
-				return true;
-			}
-			var articles = [];
-			for (var i in items) {
-				articles.push(i);
-			}
-			ZU.processDocuments(articles, scrape);
-		});
-	} else {
-		scrape(doc, url);
-	}
+function transform(ris) {
+	return ris.trim()
+		.replace(/M1\s*-/g, "IS  -") // gale puts issue numbers in M1
+		.replace(/^(?:L2|M2)\s+-.+\n/gm, '') // Ignore
+		.replace(/^SP\s+-\s+NA\n/gm, '') // Remove missing page numbers
+		.replace(/^N1(?=\s+-\s+copyright)/igm, 'CR');
+
+}
+
+function encodeString(value) {
+	return encodeURIComponent(value).replace(/%20/g, "+");
+}
+
+function getCitableDocuments(doc) {
+	return doc.getElementsByClassName('zotero');
+}
+
+return {
+	detect: detect,
+	process: process
+};
+}());
+
+function detectWeb(doc, url) { // eslint-disable-line no-unused-vars
+	return GaleZotero.detect(doc);
+}
+
+function doWeb(doc, url) { // eslint-disable-line no-unused-vars
+	return GaleZotero.process(doc);
 }
+
 /** BEGIN TEST CASES **/
+// eslint-disable-next-line no-unused-vars
```

Comment:
Remove all these eslint-disable lines. You should be using this repo's lint rules, which should handle all of this properly.